### DISCUSSION TITLE
How to handle multiple joins with select field?

### DISCUSSION THREAD

**Discussion by VaZark** [prisma/prisma#13395] [Q&A] [answered]
I'm currently building a query with multiple joins to fetch select fields across the joins. While I am certain that prisma should be able to support these, I am unable to intuitively figure out the exact query to run 

Are there any examples or docs I am refer to for multiple joins ?

For reference, one of the queries that I run look something like

 ```sql
    select
      p.programme,
      p.display_name,
      ep.is_active as entreprise_active,
      ap.is_active as agency_active,
      p.is_active as tenant_active
    from
      "Product" p
      inner join "EntrepriseProduct" ep on p.programme = ep."productId"
      inner join "AgencyProduct" ap on ep."entrepriseId" = (
          select
              a."entrepriseId"
          from
              "Agency" a
          where
              a.id = ${agencyId}
      )
      and ep."productId" = ap."productId"
```

**Comment by nurul3101** [comment]
Hey @VaZark 👋 
Could you share your "Product", "EnterpriseProduct" and "AgencyProduct" models?
On a side note you could directly invoke this query by using [Raw Query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).

**Reply by VaZark** [reply, marked as answer]
Yup, I ended up going wth Raw Queries

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
